File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Bands/BandResultArtistDto.ts
import { BandResultRoleArtistDto } from "../Bands/BandResultRoleArtistDto";

export class BandResultArtistDto {
    artistId: Number = 0;
    name: String = "";
    roleArtist: BandResultRoleArtistDto = new BandResultRoleArtistDto();
}

File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Band/BandResultArtistDto.cs
namespace ArenaGestor.APIContracts.Band
{
    public class BandResultArtistDto
    {
        public int ArtistId { get; set; }

        public string Name { get; set; }

        public BandResultRoleArtistDto RoleArtist { get; set; }
    }
}
